roborio-src/FROGlib/xbox.py

```diff
@@ -10,17 +10,17 @@ class FROGXboxDriver(XboxController):
     """Custom Xbox Controller class for the driver controller specifically
     for field-oriented swerve drive control.
     """
-    DEADBAND = 0.15
-    DEBOUNCE_PERIOD = 0.5
+
     MODE = 0  # run auto routines
 
-    def __init__(self, channel):
+    def __init__(self, channel, deadband, debouncePeriod, translationSlew, rotSlew):
         super().__init__(channel)
         self.button_latest = {}
         self.timer = Timer()
-        self.xSlew = SlewRateLimiter(1.5)
-        self.ySlew = SlewRateLimiter(1.5)
-        self.rotSlew = SlewRateLimiter(1.5)
+        self.deadband = deadband
+        self.debounce_period = debouncePeriod
+        self.translation_slew = translationSlew
```

Comment:
Here we need an xSlew and ySlew, but they get the same constant (kTranslationSlew) so that the slew rates are the same, but calculated separately.

should be like this:  
`self.xSlew = SlewRateLimiter(translationSlew)`

same idea for ySlew and rotSlew.